### DISCUSSION TITLE
CLI 32bit Windows

### DISCUSSION THREAD

**Discussion by Ddelte** [emqx/MQTTX#1400] [Q&A]
Hi, it's possibile to download a 32bit Windows version for Mqttx CLI?

Thanks
Diego

**Comment by ysfscream** [comment]
Hi @Ddelte Thanks for asking about a 32-bit Windows CLI.

Currently only a 64-bit version is offered for the MQTTX CLI on Windows. The desktop app supports 32-bit Windows but the CLI does not at this time. We appreciate your feedback and will consider a 32-bit CLI build in future if more users request it. For now it's 64-bit only.
